tests/integration/llm/prepare.py

```diff
@@ -773,6 +773,7 @@ def build_lmi_dist_model(model):
         )
     options = lmi_dist_model_list[model]
     options["engine"] = "MPI"
+    options["option.mpi_mode"] = "true"
```

Comment:
this is not needed? Given engine is MPI

Reply:
Because this change require this property: https://github.com/deepjavalibrary/djl-serving/blob/master/engines/python/setup/djl_python/rolling_batch/lmi_dist_rolling_batch.py#L42

Reply:
I got error with this.